src/components/feedback/Toast/types.ts

```diff
@@ -9,3 +9,12 @@ export interface ToastContainerProps extends Omit<ReactToastifyProps, 'transitio
    */
   transitionType?: 'Slide' | 'Bounce' | 'Zoom' | 'Flip'
 }
+
+export interface ToastifyToastOptionsCustom extends ToastifyToastOptions {
+  /**
+   * Set the delay in ms to close the toast automatically.
+   * Use `false` to prevent the toast from closing.
+   * `Default: false`
+   */
+  autoClose?: number | false
```

Comment:
`autoClose` property should be moved to `ToastContainerProps`. We don't need another interface just to override the `autoClose` prop

Reply:
Inside `ReactToastifyProps` , the interface we extends, you already have the `autoClose` property. You just need to add it in the `Omit` section and then set the default

Reply:
modified it accordingly in the latest commit